### ISSUE TITLE
test_redistance in core fails

### ISSUE THREAD

**Issue by philbucher** [KratosMultiphysics/Kratos#1092]
This is the issue for the discussion started in #1086 
the problem was the following:
```
======================================================================
FAIL: test_model_part_sub_model_parts (test_redistance.TestRedistance)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/philippb/software/Kratos_aux/kratos/tests/test_redistance.py", line 47, in test_model_part_sub_model_parts
    self.assertAlmostEqual(max_distance, 0.4473365725705744)
AssertionError: 0.44733589605275026 != 0.4473365725705744 within 7 places

----------------------------------------------------------------------
```

**Comment by philbucher** [comment]
seems like the results are random, this is from the nightly tests:
```
======================================================================
FAIL: test_model_part_sub_model_parts (test_redistance.TestRedistance)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/ubuntu/Kratos/kratos/tests/test_redistance.py", line 47, in test_model_part_sub_model_parts
    self.assertAlmostEqual(max_distance, 0.4473365725705744)
AssertionError: -1.0 != 0.4473365725705744 within 7 places

----------------------------------------------------------------------
```

**Comment by RiccardoRossi** [comment]
i will take care of this

**Comment by philbucher** [comment]
solved in #1116
